### PR TITLE
Add test documentation

### PR DESCRIPTION
**Pull request by kamicut**
Adds documentation for Playwright tests in the frontend. The information is a little dense but I tried to simplify it as much as possible. Please let me know if there is a topic you would like me to expand on because I think this could be a good internal documentation for us using Playwright @wrynearson @sunu

To merge:

1. Read the documentation
2. Try running tests
3. If any information does not make a lot of sense please let me know so I can rewrite it.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  await page.goto('/dashboard');
+  await expect(page).toHaveText('h1', 'Dashboard');
+});
+```
+
+### Writing tests
+
+**Playwright tools for writing tests**
+
+Playwright differs from other testing frameworks by using accessibility selectors to locate elements on the page. This allows us to write tests that are more resilient to changes in the DOM. When you write a test, you can use the record mechanism to record the behavior of the user on the page as well as select elements on the page to use as locators. You can learn more about VSCode here: [https://playwright.dev/docs/getting-started-vscode](https://playwright.dev/docs/getting-started-vscode)
```
Comment (frozenhelium) [inline tests/README.md:83]: Maybe, the section about VS code be separated, it looks a bit out of place at the moment.

**Review by frozenhelium — APPROVED** [review]
@kamicut This looks great! A good addition could be some more details around the file naming convention for the test (i.e. Are we creating separate file for a view? Or, a component?). Also is is a convention to add `.spec.js` in file name?

**Comment by wrynearson** [comment]
Sorry @kamicut I dropped this for a bit. Let us know next week if this is good to deploy 🚀

**Comment by kamicut** [comment]
@wrynearson yes this is good to go, it's just documentation so it shouldn't affect much in staging.

**Pull request merged**